mw-server/schemas/users.schema.go

```diff
@@ -60,3 +60,13 @@ type GetAllUsersResponse struct {
 	Size  int64                       `json:"size" validate:"required"`
 	Users []UserPlainResponseWithInfo `json:"users" validate:"required"`
 }
+
+type ChatUserPayload struct {
+	UserID string `json:"userId" validate:"required"`
+}
+
+type ChatUserResponse struct {
```

Comment:
UserPlain

Reply:
We already have that name, what do you think about BasicUserResponse?

Reply:
Done